.tree-sitter-lint/tree-sitter-lint-local/Cargo.toml

```diff
@@ -8,7 +8,7 @@ tree-sitter-lint = { path = "../.." }
 local_rules = { path = "../local_rules" }
 tree-sitter-lint-plugin-replace-foo-with = { path = "../.././tests/fixtures/tree-sitter-lint-plugin-replace-foo-with" }
 tracing = "0.1.37"
-tracing-subscriber = "0.3.17"
+tracing-subscriber = { version = "0.3.17", features = ["std", "fmt", "env-filter"] }
```

Comment:
I think this stuff just hasn't been re-generated for a while, at this point could probably un-commit it?